required/latex-lab/latex-lab-graphic.dtx

```diff
@@ -184,25 +184,15 @@
 \RequirePackage{l3opacity}
 %    \end{macrocode}
 %
-%\subsection{Saving the position}
-% We need a replacement for zref-savepos. This uses l3ref which loaded 
-% by the pdfmanagement/tagpdf.
 %
-% TODO: this is perhaps needed in other places too and should be moved to
-% l3ref!!
-%    \begin{macrocode}
-\ref_attribute_gset:nnnn{xpos}{0}{shipout}{\int_use:N\tex_lastxpos:D}
-\ref_attribute_gset:nnnn{ypos}{0}{shipout}{\int_use:N\tex_lastypos:D}
-%    \end{macrocode}
-
 % \begin{macro}{\@@_graphic_savepos:n}
 % this is the command which stores the position. It uses 
 % two savepos in case bidi needs it (see zref-savepos).
```

Comment:
maybe actually explain the reason for this rather than referring to zref-savepos